CHANGELOG.rst

```diff
@@ -7,6 +7,18 @@ All notable changes to this project after version 1.0.1 will be documented in th
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+[v2.2.0]
+--------
+This is to be considered a compatibility release for `aiida-core<2,>1.6`. It deviates from
+`master` and `develop` from this point.
```

Comment:
Maybe add some commit hashes? It is essentially on top of the last  pre-2.0 compatibility commit if my understanding is correct.